WalletWasabi.Fluent/Models/Wallets/WalletCoinsModel.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using DynamicData;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.Clustering;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Models.Wallets;
+
+[AutoInterface]
+public partial class WalletCoinsModel
+{
+	private readonly Wallet _wallet;
+
+	public WalletCoinsModel(Wallet wallet, IWalletModel walletModel)
+	{
+		_wallet = wallet;
+
+		var initialCoinList = Observable.Defer(() => GetCoins().ToObservable());
+		var initialPocketList = Observable.Defer(() => _wallet.GetPockets().ToObservable());
+		var transactionProcessed = walletModel.TransactionProcessed;
+		var anonScoreTargetChanged = walletModel.WhenAnyValue(x => x.Settings.AnonScoreTarget).ToSignal();
+		var signals = transactionProcessed.Merge(anonScoreTargetChanged);
+
+		List =
+			initialCoinList
+				.Concat(signals.SelectMany(_ => GetCoins()))
+				.ToObservableChangeSet(x => x.Key)
+				.Throttle(TimeSpan.FromMilliseconds(250), RxApp.MainThreadScheduler);
```

Comment:
Why is the throttle?

Reply:
> Why is the throttle?

@soosr that was initially intended to prevent multiple fires due to listening to several different events, but I've realized it causes other misbehaviors

Throttle removed.